### PR TITLE
Open Privacy Policy in external browser

### LINKED ISSUE #165: Link to Privacy Policy in Settings page
A link to the app's privacy policy should be added to the settings page

### PR DESCRIPTION
**Pull request by jonasanker**
Closes #165

### CODE REVIEW

**Comment by jonasanker** [comment]
I choose to open it using an external browser (instead of WebView) as the user will download a PDF and not read a webpage as such

**Review by marfavi — APPROVED** [review]
OK, I made `Privacy policy` sentence case and changed the `About Cafe Analog` section header to just say `About` (as intended).

**Review by marfavi — CHANGES_REQUESTED** [review]
Use the `launchURL` method instead (that already launches an url in an external window).

File: lib/base/strings.dart [hunk 1]
```diff
@@ -297,6 +299,7 @@ abstract class Strings {
 
   // Errors
   static const error = 'Error';
+  static const cantLaunchUrl = 'The app can not launch the Url';
```
Comment (marfavi) [inline lib/base/strings.dart:302]: How about "Couldn't find an app to open $url"?

Comment (jonasanker) [reply lib/base/strings.dart:302]: I had the same thought but the URL might not always be meaningful for a user e.g. MobilePay Deep links etc. Then what could make sense would probably be to log the errorful url

**Review by marfavi — APPROVED** [review]

**Pull request merged**